File: Java/ArmstrongNumber.java
import java.util.Scanner;

public class ArmstrongNumber {

    public static void main(String[] args) {

        int number, originalNumber, remainder, result = 0;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a number: ");
        number = scanner.nextInt();
        scanner.close();
        originalNumber = number;

        while (originalNumber != 0) {
            remainder = originalNumber % 10;
            result += Math.pow(remainder, 3);
            originalNumber /= 10;
        }

        if (result == number)
            System.out.println(number + " is an Armstrong number.");
        else
            System.out.println(number + " is not an Armstrong number.");
    }
}

File: python/Armstrong_number.py
# 153 = 1*1*1 + 5*5*5 + 3*3*3
# Enter a three-digit integer: 371
# 371 is an Armstrong number.

n=371

m=sum([int(i)**3 for i in str(n) ])
if n==m:
	print("Its Armstrong number ")
else:
	print("Its not Armstrong number")
